monors.py

```diff
@@ -487,16 +486,71 @@ def main():
 
     gh_slack_usermapping = json.load(open("monors_slack_users.json")) if slack else None
 
-    reviewers = sorted([collaborator ["login"] for collaborator in get_collaborators (gh, cfg["owner"], cfg["repo"])])
+    headers = { 'AUTHORIZATION': "token %s" % cfg ["token"], 'ACCEPT': 'application/json', 'CONTENT-TYPE': 'application/json' }
+    query = {}
+    query["query"] = '''
+            {
+                rateLimit {
+                    limit
+                    cost
+                    remaining
+                    resetAt
+                }
+                repository(owner: "mono", name: "mono") {
+                    collaborators(first: 100) {
+                        edges { node { login } }
+                    }
+                    pullRequests(last: 30, states: OPEN, orderBy: {field: UPDATED_AT, direction: ASC}) {
+                        edges {
+                            node {
+                                number
+                                mergeable
+                                title
+                                comments(last: 20) { edges { node { bodyText } } }
```

Comment:
I believe 20 comments is a tight budget. I assume this graphql query counts as one API access, right?

Reply:
Sort of. GraphQL rate limiting is a bit more complex (read more here: https://developer.github.com/v4/guides/resource-limitations/#rate-limit). I checked this query and it returns a cost of `1` so it should be fine.